routing/cross_mwm_index_graph.cpp

```diff
@@ -142,17 +147,17 @@ void CrossMwmIndexGraph::GetEdgeList(Segment const & /* s */,
   NOTIMPLEMENTED();
 }
 
-TRoutingMappingPtr CrossMwmIndexGraph::GetRoutingMapping(NumMwmId numMwmId)
+pair<TRoutingMappingPtr, unique_ptr<MappingGuard>> CrossMwmIndexGraph::GetRoutingMapping(NumMwmId numMwmId)
```

Comment:
```c++
  /// \returns routing mapping and guard for memory for succent stuctures in RoutingMapping::m_segMapping.
  /// \note The result of GetRoutingMapping() call should be kept in a local value to prevent
  /// an immediate unmaping some data after a GetRoutingMapping() call.
  std::pair<TRoutingMappingPtr, std::unique_ptr<MappingGuard>> GetRoutingMapping(NumMwmId numMwmId);
```